python/lsst/ts/MTAOS/config_schema.py

```diff
@@ -43,12 +43,61 @@
   instrument:
     description: Type of instrument for optical feedback control (OFC) to use
     type: string
-    enum: [lsst, comcam, sh, cmos]
     default: comcam
 
+  data_path:
+    description: Path to the data butler.
+    type: string
+    default: /project/shared/
+
+  run_name:
+    description: >-
+      Which name to use when running the pipeline task. This defines
+      the location where the data is written in the butler.
+    type: string
+    default: mtaos_wep
+
+  collections:
+    description: Name of the collections where the data is written in the butler.
+    type: string
+    default: refcats,LSSTComCam/raw/all,LSSTComCam/calib
+
+  pipeline_instrument:
+    description: >-
+      A dictionary that maps the name of the instrument to the name used in
+      the pipeline task.
+    anyOf:
+      - type: object
+        properties:
+          comcam:
+            type: string
+          lsstCam:
+            type: string
+          lsstFamCam:
+            type: string
+        additionalProperties: false
+      - type: "null"
+    default: null
+
+  pipeline_n_processes:
+    description: Number of processes to use when running pipeline.
+    type: integer
+    default: 9
+
+  zernike_table_name:
+    description: Name of the table in the butler with zernike coeffients.
+    type: string
+    default: zernikeEstimate
+
+  reference_detector:
+    description: Which detector to use as a reference for determining the boresight information.
+    type: integer
+    default: 0
+
 required:
   - camera
   - instrument
+  - data_path
```

Comment:
Which `data_path`? Path of `butler`? Please help to clarify this.

Reply:
The description of the fields are above: https://github.com/lsst-ts/ts_MTAOS/blob/92b1d47422219bf9bea7ab1912685b8642753bca/python/lsst/ts/MTAOS/config_schema.py#L48-L51

here I am just adding the required ones.